modules/options/src/main/scala/scala/build/actionable/ActionablePreprocessor.scala

```diff
@@ -0,0 +1,21 @@
+package scala.build.actionable
+
+import scala.build.Ops._
+import scala.build.errors.{BuildException, CompositeBuildException}
+import scala.build.options.BuildOptions
+
+object ActionablePreprocessor {
+  val actionableHandlers = Seq[ActionableHandler[_ <: ActionableDiagnostic]](
```

Comment:
NIT No need of the type constraint here (`ActionableHandler` adds it already)
```suggestion
  val actionableHandlers = Seq[ActionableHandler[_]](
```